src/guide/1/essentials/database-mappings/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: full.html.hbs
+title: Define database mapping
+order: 400
+---
+
+# Database mappings
+
+With Elasticsearch, it is possible to define mappings for collections. These mappings allows you to configure the way Elasticsearch will handle these collections.
+
+There are 3 root fields for mapping configuration:
+ - [properties]({{ site_base_path}}guide/1/essentials/database-mappings/#properties-types-definition): collection types definition
+ - [dynamic]({{ site_base_path}}guide/1/essentials/database-mappings/#dynamic-mapping-policy): dynamic mapping policy against new fields
+ - [_meta]({{ site_base_path}}guide/1/essentials/database-mappings/#collection-metadata): collection metadata
+
+The following API methods can be used to modify these mappings:
+ - [collection:create]({{ site_base_path }}api/1/controller-collection/create/)
+ - [collection:updateMapping]({{ site_base_path }}api/1/controller-collection/update-mapping/)
+
+---
+
+## Properties types definition
+
+The definition of the types of fields that will be inserted in a collection allows Elasticsearch to optimize the indexing of your data for future searches.  
```

Comment:
I'm not entirely sure about the last part of the following suggestion: untyped fields are not indexable at all, are they?

```suggestion
The field type definitions that will be inserted in a collection allow Elasticsearch to index your data for future searches.  
```